content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Istio CNI Installation
+description: Instructions for installing and using Istio with the Istio CNI plugin to allow Istio users to deploy services via lower privileged RBAC.  The Istio CNI plugin removes the requirement for the privileged, `NET_ADMIN` initialization container in the users' pods in the mesh.
+weight: 70
+keywords: [kubernetes,cni,sidecar,proxy,network,helm]
+---
+
+By default Istio injects an `initContainer`, `istio-init`, in pods deployed in the mesh.  The `istio-init`
+container sets up the pod network traffic redirection to/from the Istio sidecar proxy.  This requires the user
+or service-account deploying pods to the mesh to have high enough Kubernetes RBAC permissions to deploy
+[`NET_ADMIN` containers](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container).  Requiring Istio users to have such high Kubernetes RBAC permissions is problematic for
+some organizations' security compliance.  The Istio CNI plugin is a replacement for the `istio-init`
+container that performs the same networking functionality but without requiring Istio users to have high
+Kubernetes RBAC permissions.
+
+The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle's network
+setup phase, thereby removing this level of RBAC requirement from users deploying pods into the Istio mesh.  The
+[Istio CNI plugin](https://github.com/istio/cni) replaces the functionality provided by the `istio-init` container.
+
+## Prerequisites
+
+1.  Install Kubernetes with the container runtime supporting CNI and `kubelet`s configured
+    with the main [CNI](https://github.com/containernetworking/cni) plugin enabled via `--network-plugin=cni`.
+    *  Kubernetes installations for IBM Cloud IKS, Azure AKS, and AWS EKS clusters meet this criteria.
```

Comment:
Please alphabetize list of Kubernetes products.  I'd change the text to something like "clusters .." to "clusters have this capability."

Reply:
done